charts/datadog/README.md

```diff
@@ -135,17 +135,30 @@ The command removes all the Kubernetes components associated with the chart and
 
 ## Configuration
 
-As a best practice, a YAML file that specifies the values for the chart parameters should be provided to configure the chart:
+As a best practice, a YAML file that specifies the values for the chart parameters should be used to configure the chart. Any parameters not specified in this file will default to those set in [values.yaml](values.yaml).
 
-1. **Copy the default [`datadog-values.yaml`](values.yaml) value file.**
-2. Set the `apiKey` parameter with your [Datadog API key](https://app.datadoghq.com/account/settings#api).
-3. Upgrade the Datadog Helm chart with the new `datadog-values.yaml` file:
+1. Create an empty `datadog-values.yaml` file.
+2. Set the `apiKey` and `appKey` parameters with your [Datadog API key](https://app.datadoghq.com/organization-settings/api-keys) and [App key](https://app.datadoghq.com/organization-settings/application-keys).
+
```

Comment:
can we advice to create a secret to store the API and APP key outside the values.yaml file

```
kubectl create secret generic datadog-secret --from-literal api-key=$DD_API_KEY --from-literal app-key=$DD_APP_KEY
```

```yaml
datadog:
  apiKeyExistingSecret: datadog-secret
  appKeyExistingSecret: datadog-secret
```